ios/Handlers/LoginHandler.swift

```diff
@@ -73,7 +77,8 @@ extension LoginHandler: ONGAuthenticationDelegate {
     }
 
     func userClient(_: ONGUserClient, didReceive challenge: ONGBiometricChallenge) {
-        challenge.sender.respondWithPinFallback(for: challenge)
+        let prompt = ""
+        challenge.sender.respond(withPrompt: prompt, challenge: challenge)
```

Comment:
If you really want the prompt to be empty you can directly pass it here, no need for creating such simple constant.
```
challenge.sender.respond(withPrompt: "", challenge: challenge)